frontend/src/CalendarDropDown.js

```diff
@@ -4,12 +4,23 @@ import 'react-datepicker/dist/react-datepicker.css'
 import hu from 'date-fns/locale/hu'
 registerLocale('hu', hu)
 
-function CalendarDropDown({ date, setDate, placeholderText }) {
+function CalendarDropDown({
+  name,
+  value,
+  setFieldValue,
+  placeholderText,
+  setDate,
+  date,
+}) {
   return (
     <DatePicker
       locale="hu"
-      selected={date}
-      onChange={(date) => setDate(date)}
+      selected={setFieldValue ? (value && new Date(value)) || null : date}
+      onChange={
+        setFieldValue
```

Comment:
Mondtam, hogy ha kell, akkor wrappeljük be a CalendarDropDown-t egy Formik-os wrapperbe. Azért javasoltam ezt, mert gondoltam, hogy így elágazik majd a kód. Ha lenne egy CalendarDropDownField, akkor az átadhatná a setFieldValue-ját setDate gyanánt és ez az egész if-elgetés kiesne.

Reply:
A másik, hogy useField-et kellene abban használnunk és nem a setFieldValue-t bepasszolgatni prop-ként.